db/migrate/20220924120844_create_airlines.rb

```diff
@@ -4,6 +4,7 @@ def change
       t.string :name
       t.string :image
       t.float :price
+      t.references :user, null: false, foreign_key: true
```

Comment:
Kindly, don't change anything inside the migration files directly because it'll cause more errors in the long run. I recommend that you use commands that'll apply the appropriate changes into your migration files.